app/assets/javascripts/templates.js

```diff
@@ -0,0 +1,4 @@
+window.templates = {};
```

Comment:
We shouldn't commit this file. (It's weird)

Reply:
Removed 👍 (well left it as an empty file as before)